pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -926,3 +940,16 @@ func (r *Builder) GetPopulatorTaskName(pvc *core.PersistentVolumeClaim) (taskNam
 	err = planbase.VolumePopulatorNotSupportedError
 	return
 }
+
+func getDeviceNumber(deviceString string) int {
+	if len(deviceString) == 0 {
+		return 0
+	}
+
+	for i := len(deviceString) - 1; i >= 0; i-- {
+		if unicode.IsLetter(rune(deviceString[i])) {
+			return int(deviceString[i] - 'a' + 1)
+		}
+	}
+	return 0
```

Comment:
is it possible to reach here? if so, does it mean we didn't find what we need? info log in such case? or it's just for making a return statement then this is enough.

Reply:
umm basically not really, because the virt-v2v will fail if the string is wrong, but its just for any case, we dont want to fail the migration because of boot order, ill add a log here.